explorer-ui/src/components/Activities/Activities.jsx

```diff
@@ -18,7 +19,7 @@ const apiKey = "5ae2e3f221c38a28845f05b66afc7a4b942f1b2a702f9c54e864e3c6";
 
 export default function Activities({currCountry, currProvince, currCity, latitude, longitude, handleAttractionsSelected}) {
   const [eventData, setEventData] = useState([]);
-  const [xidList, setXIDList] = useState([]);
+  const [xid, setXID] = useState([]);
```

Comment:
nit - xids, setXIDs to indicate multiple

Reply:
no longer getting multiple xids (since now only using the xid to retrieve more information of a selected tourist attraction - there would be no need).